lib/index.js

```diff
@@ -1,66 +1,48 @@
 var canvas = document.getElementById('game');
 var context = canvas.getContext('2d');
 
+// DINOSAUR stuff --------------------------------------------------------------
+var dino_img_right = document.getElementById("dino_right");
+var dino_img_left = document.getElementById("dino_left");
+dino_img_right.style.visibility = 'hidden';
+dino_img_left.style.visibility = 'hidden';
+
+var jumpSteps = 15;
+var jumpTotal = 30;
+var jumpSize = jumpTotal/jumpSteps;
+
 function Dinosaur() {
+  this.height = 25;
+  this.width = 25;
   this.x = canvas.width / 2;
-  this.y = canvas.height - 15;
-  this.height = 15;
-  this.width = 10;
+  this.y = canvas.height - this.height;
   this.status = null;
   this.direction = "right";
+  this.count = 0;
 }
 
-function Bubble(x, y, direction){
-  this.x = x;
-  this.y = y;
-  this.direction = direction;
-  this.status = "new";
-  this.height = 5;
-  this.width = 5;
-}
-
-Bubble.prototype.draw = function(direction) {
-  context.fillRect(this.x, this.y, this.width, this.height);
-  var bubble = this;
-  var count = 0;
-  if (this.status === "new") {
-    this.status = "done";
-    if (direction === "right") {
-      var floatRight = function(that) {
-        if (count < 50 && that.x < canvas.width - 5) {
-          count++;
-          that.x++;
-          setTimeout(floatRight, 25, that);
-        } else if ((count >= 50 && that.y > 0) || (that.x === canvas.width - 5 && that.y > 0)){
-          count++;
-          that.y--;
-          setTimeout(floatRight, 25, that);
-        }
-      };
-      floatRight(bubble);
-    } else if (direction === "left") {
-      var floatLeft = function(that) {
-        if (count < 50 && that.x > 0) {
-          count++;
-          that.x--;
-          setTimeout(floatLeft, 25, that);
-        } else if ((count >= 50 && that.y > 0) || (that.x === 0 && that.y > 0)){
-          count++;
-          that.y--;
-          setTimeout(floatLeft, 25, that);
-        }
-      };
-      floatLeft(bubble);
-    }
+Dinosaur.prototype.mouthX = function() {
```

Comment:
mouthX and mouthY can also be instance variables instead of functions and have the methods .left and .right change them